### PR TITLE
Adjust README.md to prepare for Submission

### PR DESCRIPTION
**Pull request by StefanThoene**
First draft

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,19 @@ chemtrail-distribution system based on flight paths supplied by the user and the
...
+
+Our main idea when joining the Hackathon was to create something interesting and having fun.
+Here some bullet Points of our main lessons we learned: 
+
+• Collaborate on GitHub effectively 
+• Leverage pyproject.toml files for requirements
+• How to build a local virtual python environment
+• Review Pull request
+• Analyse code runtime
+• How to use numpy docstring
```
Comment (beliaev-maksim) [inline README.md:25]:
```suggestion
* Collaborate on GitHub effectively 
* Leverage pyproject.toml files for requirements
* How to build a local virtual python environment
* Review Pull request
* Analyse code runtime
* How to use numpy docstring
```

File: README.md [hunk 2]
```diff
@@ -7,3 +7,19 @@ chemtrail-distribution system based on flight paths supplied by the user and the
...
+# Installation Instruction
+
+# User Guide
+
+# What did we learn
+
+Our main idea when joining the Hackathon was to create something interesting and having fun.
+Here some bullet Points of our main lessons we learned: 
+
+• Collaborate on GitHub effectively
```
Comment (pluAtAnsys) [inline README.md:20]:
```suggestion
• Collaborate on GitHub effectively 
* Practice working with scrum and sprints during the development 
```

Comment (beliaev-maksim) [reply README.md:20]: mb practice working in agile environment ?

Comment (pluAtAnsys) [reply README.md:20]: I believe they will love to see those words :p

File: README.md [hunk 3]
```diff
@@ -9,4 +9,24 @@ energy cost in Joules, as well as an optional visual representation on a map and
...
+
+# User Guide
+
+# What did we learn
+
+Our main idea when joining the Hackathon was to create something interesting and having fun.
+Here some bullet Points of our main lessons we learned: 
+
+* How to set project goals
+* How to split task among all team members
```
Comment (beliaev-maksim) [inline README.md:24]:
```suggestion
* How to split tasks among all team members
```

**Review by beliaev-maksim — APPROVED** [review]

**Pull request merged**